src/Illuminate/Database/Schema/PostgresBuilder.php

```diff
@@ -104,8 +104,12 @@ public function dropAllTypes()
      */
     public function getAllTables()
     {
+        if (! is_array($schema = $this->connection->getConfig('schema'))) {
+            $schema = [$schema];
+        }
+
         return $this->connection->select(
-            $this->grammar->compileGetAllTables($this->connection->getConfig('schema'))
+            $this->grammar->compileGetAllTables($schema)
```

Comment:
I believe all you need to do here is to cast => `(array) $schema)`

Then you can remove even the introduced `if` condition,

Reply:
Oh, good call. Will implement.